### ISSUE TITLE
Media queries and breakpoints

### ISSUE THREAD

**Issue by reefki** [andrew-worsfold/tailor#78]
I think current media queries and breakpoints need an improvement:

```
$mobile: "only screen and (max-width: 320px)" !default;
$tablet: "only screen and (min-width: 321px) and (max-width: 720px)" !default;
$tablet-up: "only screen and (min-width: 321px)" !default;
$desktop: "only screen and (min-width: 721px)" !default;
```

I'm getting a lot of problem on tablet and mobile view, I've posted an example of the problem on #47   a screen 321px is threaten as a tablet which actually shouldn't be. As you know there are various width of mobile devices not just up to 320px, http://mydevice.io/devices/#sortSmartphones

I would recommend [Google Material Design Breakpoints](https://material.google.com/layout/responsive-ui.html#responsive-ui-breakpoints), they are recommended `600px` as max width of mobile devices.

```
$mobile: "only screen and (max-width: 600px)" !default;
$tablet: "only screen and (min-width: 601px) and (max-width: 960px)" !default;
$tablet-up: "only screen and (min-width: 601px)" !default;
$desktop: "only screen and (min-width: 961px)" !default;
```

**Comment by toderash** [comment]
+1

**Comment by atomiccherry** [comment]
+1

**Comment by andrew-worsfold** [comment]
@reefki -

I agree that nowadays devices don't neatly fall into "desktop", "tablet" or "mobile" and that the default sizes may not be the most accurate representations of those screen sizes.

The screen sizes used are those adopted by WordPress itself in the Customizer and were selected as part of an overall effort to maintain consistency with that tool.  One thing I aim to do is allow for quick and easy filtering of the device size widths so that changes feed into not just the device preview but other aspects of Tailor that reference screen size (e.g., responsive settings and custom CSS). 

There's also something else I'm working on which may render this whole discussion moot, but we'll see 😄 

Cheers,
Andrew.

**Comment by reefki** [comment]
@andrew-worsfold I agree with your effort to maintain the consistency with WordPress Customizer but that doesn't mean to ignore the real world, nowadays responsive is a critical feature and only few who still uses `320px` screen width phones.

The best solution would be make the preview iframes fixed (using WordPress Customizer preview sizes) as I mentioned in #68 and use `600px` on CSS & default `tailor_get_registered_media_queries` I believe that wouldn't break the preview on tailor page and will definitely solve all of my problems and other users too.

I appreciate your effort and times that you spent to build this awesome tool. I'm probably contribute nothing but what I'm trying to do is give you advices to make this tool even more awesome.

**Comment by fatherb** [comment]
Hi Reefki - I tried your revised settings by adjusting the _variables.scss in Assets/SCSS but they didn't do anything. Are there some other settings that need to be changed elsewhere?

**Comment by Arood** [comment]
Any news on this? I think the media query feature would be a great selling point for using Tailor more at our company. But since changes for mobile doesn't apply to most of our phones, I'm sure a lot of people would complain about it being "broken".

**Comment by genox** [comment]
+1

Breakpoints should be independently configurable to allow this gem to work nicely with themes that implement their own breakpoints. Like the ones sage provides by default. Thanks for your great work!
